third-party/CMakeLists.txt

```diff
@@ -71,9 +71,10 @@ file(COPY xxHash DESTINATION
 
 
 if(CPYTHON)
-  set(LIBPYTHONNAME libpython3.8)
-  set(LIBPYTHONNAMETAG v3.8.10)
-  set(LIBPYTHONPIP pip3.8)
+  set(TDI_PYTHON_VER 3.10)
```

Comment:
Look like TDI_PYTHON_VER is a new environment variable. How do you plan to use the TDI_PYTHON_VER?

Reply:
Yes, adding the new env variable. This can be used by repositories using target-utils as a third-party refpoint to find out what version of python is being used and use it in their cmake accordingly